composer.json

```
@@ -42,7 +43,8 @@
         "symfony/http-kernel": "~6",
         "symfony/monolog-bridge": "~6",
         "symfony/property-access": "~6",
-        "symfony/property-info": "~6"
+        "symfony/property-info": "~6",
+        "web-token/jwt-signature-algorithm-hmac": "^4.0"
```

This package is marked as abandoned.
As I understand package `web-token/jwt-framework` can be used instead. 
```suggestion
        "web-token/jwt-framework": "^3.4"
```
`suggest` section should be fixed accordingly.